sre/kvStorage_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import "testing"
+
+func TestPutandGet(t *testing.T) {
```

Comment:
testGetWhenKeyNotFound -> should return an error.
you can assert.ErrorContains(t, err, "Value not found")